tox.ini

```diff
@@ -11,7 +11,7 @@ addopts =
     --cov=datalad_crawler
     # Explicitly setting the path to the coverage config file is necessary due
     # to some tests spawning subprocesses with changed working directories.
-    --cov-config=tox.ini
+    --cov-config={toxinidir}/tox.ini
```

Comment:
@jwodder what did I do wrong? Seems not replaced

Reply:
Right, as I said above, `{toxinidir}` only takes effect in sections read & used by tox, but this is in a pytest section.

Reply:
eh -- any ideas on how to overcome this problem?